example/tb.cpp

```diff
@@ -56,13 +56,13 @@ int main(int argc, char **argv, char **env) {
     "gpio.SW14",
     "gpio.SW15",
   };
-  int num_inputs = 16;
+  int num_inputs = sizeof(inputs) / sizeof(inputs[0]);
```

Comment:
Does this require all strings to be of equal length?

Reply:
Ohh.... using sizeof on an array gets the size of the index pointers. Is that so?

Reply:
total size / sizeof of one element gives number in a array. Assuming no padding, but were are talking pointers here